go.mod

```diff
@@ -147,7 +147,7 @@ require (
 	github.com/exponent-io/jsonpath v0.0.0-20210407135951-1de76d718b3f // indirect
 	github.com/fluxcd/notification-controller/api v0.32.1
```

Comment:
@foot can we also bump this one to `github.com/fluxcd/notification-controller/api v1.0.0-rc.3`? Not sure why this is not shown along with the other Flux imports, perhaps we can move it up too.

Reply:
Oops! Good catch